torch/_torch_docs.py

```diff
@@ -4050,6 +4050,35 @@ def merge_dicts(*dicts):
             [5, 6, 7, 8]])
 """.format(**common_args))
 
+add_docstr(torch.unflatten,
+           r"""
+unflatten(input, dim, sizes) -> Tensor
+
+Expands a dimension of the input tensor over multiple dimensions.
+
+A tensor reshaping function that does the opposite of :func:`torch.flatten`.
+
+Args:
+    {input}
+    dim (int): Dimension to be unflattened, specified as an index into
+         ``input.shape``.
+    sizes (Tuple[int]): New shape of the unflattened dimension.
+         One of its elements can be `-1` in which case the corresponding output
+         dimension is inferred. Otherwise, the product of ``sizes`` *must*
+         equal ``input.shape[dim]``.
+
+Returns:
+    A ``View`` of ``input`` with the specified dimension unflattened.
+
+Examples:
```

Comment:
I don't know why the examples are being rendered funny in the docs. I think you may need to write
```suggestion
Examples::

```
See how it's rendered at the moment (taken from the message from facebook-github-bot) https://docs-preview.pytorch.org/81399/generated/torch.unflatten.html?highlight=unflatten#torch.unflatten